src/Core/Core.Tests/ScalarSupportTests.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using ChilliCream.Testing;
+using HotChocolate.Execution;
+using HotChocolate.Language;
+using HotChocolate.Types;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace HotChocolate
+{
+    public class ScalarSupportTests
```

Comment:
Could we move this test class to Types.Tests/Types/Scalars since this is more about the type system.

Reply:
Yes sir, I will move the tests and the snapshots over

Reply:
Once the ci build is completed I will push that one to the master branch.